includes/Shopping/Shopify_Query.php

```diff
@@ -297,4 +304,35 @@ public function get_search( string $search_term ) {
 
 		return $results;
 	}
+
+	/**
+	 * Parse sort type into array for product query
+	 *
+	 * @since 1.21.0
+	 *
+	 * @param string $sort_by sort condition for product query.
+	 * @return array
+	 */
+	protected function parse_sort_by( string $sort_by ): array {
```

Comment:
If this is shared logic, we should maybe create a abstance class that both these classes use.

Reply:
The resulting array keys ends up being different ---  but yes as far as the duplication (would be nice to avoid for maintenance)

Reply:
Should be part of the interface?

Reply:
Interfaces are for public methods, not protected/private methods that are specific to each implementation.

But I propose not doing this at all, see https://github.com/GoogleForCreators/web-stories-wp/pull/11552/files#r880339767